docs/sharing-role-cabalities-capablity-set.puml

```diff
@@ -0,0 +1,55 @@
+@startuml
+title sd book room
+
+actor Guest
+participant Catalog
+participant System
+participant Booking
+participant Room
+participant Payment
+
+activate Guest
+Guest -> Catalog : searchRoom(price, style)
+activate Catalog
+Catalog --> Guest : return roomList
+deactivate Catalog
+
+Guest -> System : selectRoom()
+activate System
+
+alt roomStatus = available
+    System --> Guest : ready to book
+    Guest -> System : createBooking(room)
+    System -> Booking : create
+    activate Booking
+    Booking -> Room : getBookingPrice()
```

Comment:
did correct file commit? if it is sequence diagram need to change variable names to our application area